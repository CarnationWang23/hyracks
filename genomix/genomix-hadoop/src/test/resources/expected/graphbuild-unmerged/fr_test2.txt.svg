<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.26.3 (20100126.1600)
 -->
<!-- Title: fr_test2_txt Pages: 1 -->
<svg width="888pt" height="428pt"
 viewBox="0.00 0.00 888.00 428.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph1" class="graph" transform="scale(1 1) rotate(0) translate(4 424)">
<title>fr_test2_txt</title>
<polygon fill="white" stroke="white" points="-4,5 -4,-424 885,-424 885,5 -4,5"/>
<g id="graph2" class="cluster"><title>cluster_legend</title>
<polygon fill="none" stroke="black" points="49,-8 49,-209 211,-209 211,-8 49,-8"/>
<text text-anchor="middle" x="130" y="-192.4" font-family="Times Roman,serif" font-size="14.00">legend</text>
</g>
<g id="graph3" class="cluster"><title>cluster_1</title>
<polygon fill="none" stroke="black" points="8,-217 8,-305 500,-305 500,-217 8,-217"/>
</g>
<g id="graph4" class="cluster"><title>cluster_3</title>
<polygon fill="none" stroke="black" points="8,-324 8,-412 500,-412 500,-324 8,-324"/>
</g>
<g id="graph5" class="cluster"><title>cluster_2</title>
<polygon fill="none" stroke="black" points="398,-121 398,-209 872,-209 872,-121 398,-121"/>
</g>
<!-- legend_0_0 -->
<g id="node2" class="node"><title>legend_0_0</title>
<ellipse fill="black" stroke="black" cx="59" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_0 -->
<g id="node3" class="node"><title>legend_1_0</title>
<ellipse fill="black" stroke="black" cx="201" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_0&#45;&gt;legend_1_0 -->
<g id="edge3" class="edge"><title>legend_0_0&#45;&gt;legend_1_0</title>
<path fill="none" stroke="#dd1e2f" d="M61.0074,-157C74.8673,-157 156.744,-157 188.46,-157"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="188.862,-160.5 198.862,-157 188.861,-153.5 188.862,-160.5"/>
<text text-anchor="middle" x="130" y="-162.4" font-family="Times Roman,serif" font-size="14.00">FF</text>
</g>
<!-- legend_0_1 -->
<g id="node5" class="node"><title>legend_0_1</title>
<ellipse fill="black" stroke="black" cx="59" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_1 -->
<g id="node6" class="node"><title>legend_1_1</title>
<ellipse fill="black" stroke="black" cx="201" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_1&#45;&gt;legend_1_1 -->
<g id="edge5" class="edge"><title>legend_0_1&#45;&gt;legend_1_1</title>
<path fill="none" stroke="#ebb035" d="M61.0074,-116C74.8673,-116 156.744,-116 188.46,-116"/>
<polygon fill="#ebb035" stroke="#ebb035" points="188.862,-119.5 198.862,-116 188.861,-112.5 188.862,-119.5"/>
<text text-anchor="middle" x="130" y="-121.4" font-family="Times Roman,serif" font-size="14.00">FR</text>
</g>
<!-- legend_0_2 -->
<g id="node8" class="node"><title>legend_0_2</title>
<ellipse fill="black" stroke="black" cx="59" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_2 -->
<g id="node9" class="node"><title>legend_1_2</title>
<ellipse fill="black" stroke="black" cx="201" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_2&#45;&gt;legend_1_2 -->
<g id="edge7" class="edge"><title>legend_0_2&#45;&gt;legend_1_2</title>
<path fill="none" stroke="#06a2cb" d="M61.0074,-75C74.8673,-75 156.744,-75 188.46,-75"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="188.862,-78.5001 198.862,-75 188.861,-71.5001 188.862,-78.5001"/>
<text text-anchor="middle" x="130" y="-80.4" font-family="Times Roman,serif" font-size="14.00">RF</text>
</g>
<!-- legend_0_3 -->
<g id="node11" class="node"><title>legend_0_3</title>
<ellipse fill="black" stroke="black" cx="59" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_3 -->
<g id="node12" class="node"><title>legend_1_3</title>
<ellipse fill="black" stroke="black" cx="201" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_3&#45;&gt;legend_1_3 -->
<g id="edge9" class="edge"><title>legend_0_3&#45;&gt;legend_1_3</title>
<path fill="none" stroke="#218559" d="M61.0074,-34C74.8673,-34 156.744,-34 188.46,-34"/>
<polygon fill="#218559" stroke="#218559" points="188.862,-37.5001 198.862,-34 188.861,-30.5001 188.862,-37.5001"/>
<text text-anchor="middle" x="130" y="-39.4" font-family="Times Roman,serif" font-size="14.00">RR</text>
</g>
<!-- 1,1 -->
<g id="node15" class="node"><title>1,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-253 37,-267 82,-267 82,-253 37,-253"/>
<text text-anchor="start" x="42" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AAACG</text>
<polygon fill="#218559" stroke="#218559" points="37,-239 37,-253 82,-253 82,-239 37,-239"/>
<text text-anchor="start" x="44.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CGTTT</text>
</g>
<!-- 1,2 -->
<g id="node16" class="node"><title>1,2</title>
<ellipse fill="none" stroke="black" cx="201" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-253 179,-267 224,-267 224,-253 179,-253"/>
<text text-anchor="start" x="185" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AACGT</text>
<polygon fill="#218559" stroke="#218559" points="179,-239 179,-253 224,-253 224,-239 179,-239"/>
<text text-anchor="start" x="186" y="-243.667" font-family="Times Roman,serif" font-size="10.00">ACGTT</text>
</g>
<!-- 1,1&#45;&gt;1,2 -->
<g id="edge30" class="edge"><title>1,1&#45;&gt;1,2</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-254.755C116.207,-254.208 132.729,-254.105 148.049,-254.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-257.957 158.425,-254.756 148.534,-250.96 148.326,-257.957"/>
</g>
<!-- 1,2&#45;&gt;1,1 -->
<g id="edge34" class="edge"><title>1,2&#45;&gt;1,1</title>
<path fill="none" stroke="#218559" d="M158.425,-267.244C143.825,-267.792 127.305,-267.895 111.982,-267.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-264.043 101.605,-267.245 111.497,-271.04 111.704,-264.043"/>
</g>
<!-- 1,3 -->
<g id="node17" class="node"><title>1,3</title>
<ellipse fill="none" stroke="black" cx="325" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-253 303,-267 348,-267 348,-253 303,-253"/>
<text text-anchor="start" x="309.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">ACGTA</text>
<polygon fill="#218559" stroke="#218559" points="303,-239 303,-253 348,-253 348,-239 303,-239"/>
<text text-anchor="start" x="310" y="-243.667" font-family="Times Roman,serif" font-size="10.00">TACGT</text>
</g>
<!-- 1,2&#45;&gt;1,3 -->
<g id="edge32" class="edge"><title>1,2&#45;&gt;1,3</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-254.562C252.601,-254.258 262.531,-254.176 272.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-257.822 282.443,-254.554 272.527,-250.823 272.364,-257.822"/>
</g>
<!-- 1,3&#45;&gt;1,2 -->
<g id="edge42" class="edge"><title>1,3&#45;&gt;1,2</title>
<path fill="none" stroke="#218559" d="M282.443,-267.446C273.158,-267.745 263.225,-267.824 253.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-264.175 243.327,-267.438 253.242,-271.173 253.407,-264.175"/>
</g>
<!-- 1,4 -->
<g id="node18" class="node"><title>1,4</title>
<ellipse fill="none" stroke="black" cx="449" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-253 427,-267 472,-267 472,-253 427,-253"/>
<text text-anchor="start" x="434.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">CGTAT</text>
<polygon fill="#218559" stroke="#218559" points="427,-239 427,-253 472,-253 472,-239 427,-239"/>
<text text-anchor="start" x="433.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">ATACG</text>
</g>
<!-- 1,3&#45;&gt;1,4 -->
<g id="edge38" class="edge"><title>1,3&#45;&gt;1,4</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-254.562C376.601,-254.258 386.531,-254.176 396.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-257.822 406.443,-254.554 396.527,-250.823 396.364,-257.822"/>
</g>
<!-- 3,4 -->
<g id="node23" class="node"><title>3,4</title>
<ellipse fill="none" stroke="black" cx="449" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">3,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-360 427,-374 472,-374 472,-360 427,-360"/>
<text text-anchor="start" x="433.5" y="-364.667" font-family="Times Roman,serif" font-size="10.00">ATACG</text>
<polygon fill="#218559" stroke="#218559" points="427,-346 427,-360 472,-360 472,-346 427,-346"/>
<text text-anchor="start" x="434.5" y="-350.667" font-family="Times Roman,serif" font-size="10.00">CGTAT</text>
</g>
<!-- 1,3&#45;&gt;3,4 -->
<g id="edge40" class="edge"><title>1,3&#45;&gt;3,4</title>
<path fill="none" stroke="#ebb035" d="M355.017,-286.902C371.799,-301.383 392.887,-319.58 410.832,-335.065"/>
<polygon fill="#ebb035" stroke="#ebb035" points="408.972,-338.083 418.83,-341.966 413.545,-332.783 408.972,-338.083"/>
</g>
<!-- 2,1 -->
<g id="node25" class="node"><title>2,1</title>
<ellipse fill="none" stroke="black" cx="449" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-157 427,-171 472,-171 472,-157 427,-157"/>
<text text-anchor="start" x="434.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">CGTAT</text>
<polygon fill="#218559" stroke="#218559" points="427,-143 427,-157 472,-157 472,-143 427,-143"/>
<text text-anchor="start" x="433.5" y="-147.667" font-family="Times Roman,serif" font-size="10.00">ATACG</text>
</g>
<!-- 1,3&#45;&gt;2,1 -->
<g id="edge36" class="edge"><title>1,3&#45;&gt;2,1</title>
<path fill="none" stroke="#dd1e2f" d="M358.345,-237.712C373.456,-227.157 388,-217 388,-217 388,-217 398.622,-207.945 411.002,-197.392"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="413.518,-199.846 418.857,-190.695 408.977,-194.519 413.518,-199.846"/>
</g>
<!-- 1,4&#45;&gt;1,3 -->
<g id="edge48" class="edge"><title>1,4&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M406.443,-267.446C397.158,-267.745 387.225,-267.824 377.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-264.175 367.327,-267.438 377.242,-271.173 377.407,-264.175"/>
</g>
<!-- 3,3 -->
<g id="node22" class="node"><title>3,3</title>
<ellipse fill="none" stroke="black" cx="325" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">3,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-360 303,-374 348,-374 348,-360 303,-360"/>
<text text-anchor="start" x="310" y="-364.667" font-family="Times Roman,serif" font-size="10.00">AATAC</text>
<polygon fill="#218559" stroke="#218559" points="303,-346 303,-360 348,-360 348,-346 303,-346"/>
<text text-anchor="start" x="311.5" y="-350.667" font-family="Times Roman,serif" font-size="10.00">GTATT</text>
</g>
<!-- 1,4&#45;&gt;3,3 -->
<g id="edge46" class="edge"><title>1,4&#45;&gt;3,3</title>
<path fill="none" stroke="#ebb035" d="M418.83,-287.034C402.021,-301.538 380.93,-319.738 363.004,-335.206"/>
<polygon fill="#ebb035" stroke="#ebb035" points="360.301,-332.915 355.017,-342.098 364.875,-338.215 360.301,-332.915"/>
</g>
<!-- 2,2 -->
<g id="node26" class="node"><title>2,2</title>
<ellipse fill="none" stroke="black" cx="573" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="553.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="551,-157 551,-171 596,-171 596,-157 551,-157"/>
<text text-anchor="start" x="559.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">GTATT</text>
<polygon fill="#218559" stroke="#218559" points="551,-143 551,-157 596,-157 596,-143 551,-143"/>
<text text-anchor="start" x="558" y="-147.667" font-family="Times Roman,serif" font-size="10.00">AATAC</text>
</g>
<!-- 1,4&#45;&gt;2,2 -->
<g id="edge44" class="edge"><title>1,4&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M480.611,-236.527C496.521,-224.21 515.976,-209.147 532.924,-196.027"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="535.251,-198.652 541.015,-189.762 530.965,-193.117 535.251,-198.652"/>
</g>
<!-- 3,1 -->
<g id="node20" class="node"><title>3,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">3,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-360 37,-374 82,-374 82,-360 37,-360"/>
<text text-anchor="start" x="42.5" y="-364.667" font-family="Times Roman,serif" font-size="10.00">GGAAT</text>
<polygon fill="#218559" stroke="#218559" points="37,-346 37,-360 82,-360 82,-346 37,-346"/>
<text text-anchor="start" x="44.5" y="-350.667" font-family="Times Roman,serif" font-size="10.00">ATTCC</text>
</g>
<!-- 3,2 -->
<g id="node21" class="node"><title>3,2</title>
<ellipse fill="none" stroke="black" cx="201" cy="-368" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-379.167" font-family="Times Roman,serif" font-size="10.00">3,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-360 179,-374 224,-374 224,-360 179,-360"/>
<text text-anchor="start" x="185.5" y="-364.667" font-family="Times Roman,serif" font-size="10.00">GAATA</text>
<polygon fill="#218559" stroke="#218559" points="179,-346 179,-360 224,-360 224,-346 179,-346"/>
<text text-anchor="start" x="188" y="-350.667" font-family="Times Roman,serif" font-size="10.00">TATTC</text>
</g>
<!-- 3,1&#45;&gt;3,2 -->
<g id="edge50" class="edge"><title>3,1&#45;&gt;3,2</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-361.755C116.207,-361.208 132.729,-361.105 148.049,-361.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-364.957 158.425,-361.756 148.534,-357.96 148.326,-364.957"/>
</g>
<!-- 3,2&#45;&gt;3,1 -->
<g id="edge54" class="edge"><title>3,2&#45;&gt;3,1</title>
<path fill="none" stroke="#218559" d="M158.425,-374.244C143.825,-374.792 127.305,-374.895 111.982,-374.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-371.043 101.605,-374.245 111.497,-378.04 111.704,-371.043"/>
</g>
<!-- 3,2&#45;&gt;3,3 -->
<g id="edge52" class="edge"><title>3,2&#45;&gt;3,3</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-361.562C252.601,-361.258 262.531,-361.176 272.159,-361.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-364.822 282.443,-361.554 272.527,-357.823 272.364,-364.822"/>
</g>
<!-- 3,3&#45;&gt;1,4 -->
<g id="edge58" class="edge"><title>3,3&#45;&gt;1,4</title>
<path fill="none" stroke="#ebb035" d="M353.714,-341.169C369.535,-326.385 386,-311 386,-311 386,-311 396.801,-302.428 409.459,-292.382"/>
<polygon fill="#ebb035" stroke="#ebb035" points="411.843,-294.958 417.5,-286 407.491,-289.475 411.843,-294.958"/>
</g>
<!-- 3,3&#45;&gt;3,2 -->
<g id="edge60" class="edge"><title>3,3&#45;&gt;3,2</title>
<path fill="none" stroke="#218559" d="M282.443,-374.446C273.158,-374.745 263.225,-374.824 253.602,-374.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-371.175 243.327,-374.438 253.242,-378.173 253.407,-371.175"/>
</g>
<!-- 3,3&#45;&gt;3,4 -->
<g id="edge56" class="edge"><title>3,3&#45;&gt;3,4</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-361.562C376.601,-361.258 386.531,-361.176 396.159,-361.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-364.822 406.443,-361.554 396.527,-357.823 396.364,-364.822"/>
</g>
<!-- 3,4&#45;&gt;1,3 -->
<g id="edge62" class="edge"><title>3,4&#45;&gt;1,3</title>
<path fill="none" stroke="#ebb035" d="M418.83,-341.966C402.021,-327.462 380.93,-309.262 363.004,-293.794"/>
<polygon fill="#ebb035" stroke="#ebb035" points="364.875,-290.785 355.017,-286.902 360.301,-296.085 364.875,-290.785"/>
</g>
<!-- 3,4&#45;&gt;3,3 -->
<g id="edge64" class="edge"><title>3,4&#45;&gt;3,3</title>
<path fill="none" stroke="#218559" d="M406.443,-374.446C397.158,-374.745 387.225,-374.824 377.602,-374.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-371.175 367.327,-374.438 377.242,-378.173 377.407,-371.175"/>
</g>
<!-- 2,1&#45;&gt;1,3 -->
<g id="edge16" class="edge"><title>2,1&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M417.015,-189.762C401.051,-202.122 381.587,-217.19 364.68,-230.28"/>
<polygon fill="#218559" stroke="#218559" points="362.376,-227.637 356.611,-236.527 366.661,-233.173 362.376,-227.637"/>
</g>
<!-- 2,1&#45;&gt;2,2 -->
<g id="edge14" class="edge"><title>2,1&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M491.327,-158.562C500.601,-158.258 510.531,-158.176 520.159,-158.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="520.364,-161.822 530.443,-158.554 520.527,-154.823 520.364,-161.822"/>
</g>
<!-- 2,2&#45;&gt;1,4 -->
<g id="edge20" class="edge"><title>2,2&#45;&gt;1,4</title>
<path fill="none" stroke="#218559" d="M544.286,-191.831C528.465,-206.615 512,-222 512,-222 512,-222 503.356,-227.351 492.471,-234.089"/>
<polygon fill="#218559" stroke="#218559" points="490.477,-231.207 483.817,-239.447 494.162,-237.159 490.477,-231.207"/>
</g>
<!-- 2,2&#45;&gt;2,1 -->
<g id="edge22" class="edge"><title>2,2&#45;&gt;2,1</title>
<path fill="none" stroke="#218559" d="M530.443,-171.446C521.158,-171.745 511.225,-171.824 501.602,-171.681"/>
<polygon fill="#218559" stroke="#218559" points="501.407,-168.175 491.327,-171.438 501.242,-175.173 501.407,-168.175"/>
</g>
<!-- 2,3 -->
<g id="node27" class="node"><title>2,3</title>
<ellipse fill="none" stroke="black" cx="697" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="677.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="675,-157 675,-171 720,-171 720,-157 675,-157"/>
<text text-anchor="start" x="684" y="-161.667" font-family="Times Roman,serif" font-size="10.00">TATTC</text>
<polygon fill="#218559" stroke="#218559" points="675,-143 675,-157 720,-157 720,-143 675,-143"/>
<text text-anchor="start" x="681.5" y="-147.667" font-family="Times Roman,serif" font-size="10.00">GAATA</text>
</g>
<!-- 2,2&#45;&gt;2,3 -->
<g id="edge18" class="edge"><title>2,2&#45;&gt;2,3</title>
<path fill="none" stroke="#dd1e2f" d="M615.327,-158.562C624.601,-158.258 634.531,-158.176 644.159,-158.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="644.364,-161.822 654.443,-158.554 644.527,-154.823 644.364,-161.822"/>
</g>
<!-- 2,3&#45;&gt;2,2 -->
<g id="edge26" class="edge"><title>2,3&#45;&gt;2,2</title>
<path fill="none" stroke="#218559" d="M654.443,-171.446C645.158,-171.745 635.225,-171.824 625.602,-171.681"/>
<polygon fill="#218559" stroke="#218559" points="625.407,-168.175 615.327,-171.438 625.242,-175.173 625.407,-168.175"/>
</g>
<!-- 2,4 -->
<g id="node28" class="node"><title>2,4</title>
<ellipse fill="none" stroke="black" cx="821" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="801.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="799,-157 799,-171 844,-171 844,-157 799,-157"/>
<text text-anchor="start" x="806.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">ATTCC</text>
<polygon fill="#218559" stroke="#218559" points="799,-143 799,-157 844,-157 844,-143 799,-143"/>
<text text-anchor="start" x="804.5" y="-147.667" font-family="Times Roman,serif" font-size="10.00">GGAAT</text>
</g>
<!-- 2,3&#45;&gt;2,4 -->
<g id="edge24" class="edge"><title>2,3&#45;&gt;2,4</title>
<path fill="none" stroke="#dd1e2f" d="M739.327,-158.562C748.601,-158.258 758.531,-158.176 768.159,-158.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="768.364,-161.822 778.443,-158.554 768.527,-154.823 768.364,-161.822"/>
</g>
<!-- 2,4&#45;&gt;2,3 -->
<g id="edge28" class="edge"><title>2,4&#45;&gt;2,3</title>
<path fill="none" stroke="#218559" d="M778.443,-171.446C769.158,-171.745 759.225,-171.824 749.602,-171.681"/>
<polygon fill="#218559" stroke="#218559" points="749.407,-168.175 739.327,-171.438 749.242,-175.173 749.407,-168.175"/>
</g>
</g>
</svg>
