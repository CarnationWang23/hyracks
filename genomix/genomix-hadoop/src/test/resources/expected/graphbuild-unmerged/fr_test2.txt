((2,1)	[(2,2)]	[]	[]	[(1,3)]	CGTAT)	(null)
((2,2)	[(2,3)]	[]	[]	[(1,4),(2,1)]	GTATT)	(null)
((2,3)	[(2,4)]	[]	[]	[(2,2)]	TATTC)	(null)
((2,4)	[]	[]	[]	[(2,3)]	ATTCC)	(null)
((1,1)	[(1,2)]	[]	[]	[]	AAACG)	(null)
((1,2)	[(1,3)]	[]	[]	[(1,1)]	AACGT)	(null)
((1,3)	[(2,1),(1,4)]	[(3,4)]	[]	[(1,2)]	ACGTA)	(null)
((1,4)	[(2,2)]	[(3,3)]	[]	[(1,3)]	CGTAT)	(null)
((3,1)	[(3,2)]	[]	[]	[]	GGAAT)	(null)
((3,2)	[(3,3)]	[]	[]	[(3,1)]	GAATA)	(null)
((3,3)	[(3,4)]	[(1,4)]	[]	[(3,2)]	AATAC)	(null)
((3,4)	[]	[(1,3)]	[]	[(3,3)]	ATACG)	(null)
